event_sourcery_esdb/outbox.py

```diff
@@ -153,11 +155,12 @@ def outbox_entries(
             return (
                 (
                     cast(int, entry.commit_position),
-                    dto.raw_event(entry),
+                    raw_event,
```

Comment:
Nice trick, you caught me off guard with walrus operator. However, when I look at this gen expression, it's bit too complex to me. It'd be good to refactor it in the future